docs/source/yelpsoa_configs.rst

```diff
@@ -254,41 +250,16 @@ These options are only applicable to tasks scheduled on Kubernetes.
 
 For more information on selector operators, see the official Kubernetes
 documentation on `node affinities
-<https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity>`_.
+<https://kubernetes.io/docs/concepts/configuration/assign-Pod-node/#node-affinity>`_.
 
-  * ``pod_management_policy``: An option for applications managed with `StatefulSets <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`_ to determine if the pods are managed in parallel or in order.
+  * ``Pod_management_policy``: An option for applications managed with `StatefulSets <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`_ to determine if the Pods are managed in parallel or in order.
 
-    The default value is `OrderedReady <https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#orderedready-pod-management>`_.
-    It can be set to `Parallel <https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#parallel-pod-management>`_. For example::
+    The default value is `OrderedReady <https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#orderedready-Pod-management>`_.
+    It can be set to `Parallel <https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#parallel-Pod-management>`_. For example::
 
-      pod_management_policy: Parallel
+      Pod_management_policy: Parallel
```

Comment:
i think we might have been a bit over-zealous with the find-replace here too - i think the only casing change we want to keep is `Pods are managed`